### PR TITLE
Increase `shm` to mitigate bus error

### PR DESCRIPTION
**Pull request by Alan32Liu**
To fix `Bus error (core dumped)` discovered in #1436.

### CODE REVIEW

**Review by oliverchang — APPROVED** [review]
LGTM, but I'll let @jonathanmetzman review too if there's any places were this is either not necessary to be added or missing completely.

File: experiment/resources/dispatcher-startup-script-template.sh
Comment (jonathanmetzman) [inline experiment/resources/dispatcher-startup-script-template.sh:29]: Fuzzing isn't done in this image btw. What's the default here? Because this instance usually has hundreds of GB of RAM

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Comment by jonathanmetzman** [comment]
Awesome!

**Review by oliverchang — APPROVED** [review]

File: experiment/resources/dispatcher-startup-script-template.sh
Comment (Alan32Liu) [reply experiment/resources/dispatcher-startup-script-template.sh]: Thanks! Yep, I can confirm that removing this won't matter.
I am not sure how to check the default instance, though. Where is the value of `docker_registry` defined?

Comment (oliverchang) [reply experiment/resources/dispatcher-startup-script-template.sh]: I think the default value is 64mb on my local machine. If this doesn't any fuzzing and hasn't caused any problems, we can just skip this one.

**Comment by jonathanmetzman** [comment]
Merge this when ready dongge

**Pull request merged**
